src/method.jl

```diff
@@ -27,13 +27,23 @@ with [`CodeTracking.definition`](https://github.com/timholy/CodeTracking.jl).
 The dictionary of components returned by `signature` match those returned by
 [`splitdef`](@ref) and include all that are required by [`combinedef`](@ref), except for
 the `:body` component.
+
+# keywords
+
+ - `extra_hygiene=false`: if set to `true` this forces name-hygine on the `TypeVar`s in 
```

Comment:
```suggestion
 - `extra_hygiene=false`: if set to `true` this forces name-hygiene on the `TypeVar`s in 
```

Reply:
I may have copied that from a earlier revision